db/change_cache.go

```diff
@@ -663,6 +667,7 @@ func (c *changeCache) processEntry(change *LogEntry) base.Set {
 		// There's a missing sequence (or several), so put this one on ice until it arrives:
 		heap.Push(&c.pendingLogs, change)
 		numPending := len(c.pendingLogs)
+		c.context.DbStats.StatsCache().Set(base.StatKeyPendingSeqLen, base.ExpvarIntVal(len(c.pendingLogs)))
```

Comment:
Dunno if this is any more/less costly, but any benefit to just reusing `numPending` from the line above rather than calling `len()` again?